### PR TITLE
Fixed sphinx documentation for Windows.

### PR DESCRIPTION
**Pull request by adeas31**
Copy profiler files to build/html as well so that the links work.
Only use the generated API interface. Easy to maintain.
Fixed paths for gnuplot.
Added readme with instructions on what is needed for compilation on Windows.

### CODE REVIEW

File: UsersGuide/source/scripting_api.rst [hunk 1]
```diff
@@ -30,51 +30,7 @@ OpenModelica Modelica Scripting Commands
...
-* `dirName`_
-* `readFile`_
-* `runScript`_
-* `setModelicaPath`_
-* `system`_
-* `writeFile`_
-
-All OpenModelica API Calls
+OpenModelica API Calls
 ##########################
```
Comment (sjoelund) [inline UsersGuide/source/scripting_api.rst]: Need to have exactly the same number of characters as the above line

File: UsersGuide/Makefile [hunk 2]
```diff
@@ -26,34 +26,35 @@ ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) sou
 # the i18n builder cannot share the environment and doctrees with the others
 I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) source
 
-.PHONY: help clean html dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck doctest coverage gettext readme
+.PHONY: help clean html htmlmathpng dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck doctest coverage gettext readme
 
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
-	@echo "  html       to make standalone HTML files"
```
Comment (sjoelund) [inline UsersGuide/Makefile]: Did you mess with line endings?

File: UsersGuide/source/scripting_api.rst (hunk 1 above)
Comment (adeas31) [inline UsersGuide/source/scripting_api.rst]: Why? I just removed the hard coded API list. We don't need it, right?

Comment (sjoelund) [inline UsersGuide/source/scripting_api.rst]: Note that the line note is on line 34. The line you didn't change...

File: UsersGuide/Makefile (hunk 2 above)
Comment (adeas31) [inline UsersGuide/Makefile]: What line endings should I use? I have LF.

File: UsersGuide/source/scripting_api.rst (hunk 1 above)
Comment (adeas31) [inline UsersGuide/source/scripting_api.rst]: Ahhh ok.

Comment (adeas31) [inline UsersGuide/source/scripting_api.rst]: Btw does that make any difference?

**Pull request merged**